### PR TITLE
[SP-6341] 

### PR DESCRIPTION
**Pull request by InbarPerry**

### CODE REVIEW

File: steps/greynoise/gnql-query/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,834 @@
...
+    that brings our users and customers sufficient value will eventually be transitioned
+    into it''s own individual offering.
+
+    More info: https://developer.greynoise.io/reference/gnql-1#gnqlquery-1'
+  name: gnql_query
+  type: remediation
+  vendor: greynoise
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/gnql-query/manifest.yaml:36]:
```suggestion
- description: A JSON object containing the response from the GreyNoise API.
```

File: steps/greynoise/gnql-query/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,834 @@
+apiVersion: stackpulse.io/v1
```
Comment (hagaishapira) [inline steps/greynoise/gnql-query/manifest.yaml:1]: The fact that the yaml turns out this way makes it very difficult to read. I don't understand why this happened, I thought dicts are ordered in python3.6+

File: steps/greynoise/gnql-stats/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+integrations:
+- greynoise_token
+kind: Step
+metadata:
+  description: 'Get aggregate statistics for the top organizations, actors, tags,
+    ASNs, countries, classifications, and operating systems of all the results of
+    a given GNQL query.
+
+    More info: https://developer.greynoise.io/reference/gnql-1#gnqlstats-1'
+  name: gnql_stats
```
Comment (hagaishapira) [inline steps/greynoise/gnql-stats/manifest.yaml:22]:
```suggestion
  name: greynoise_get_gnql_stats
```

File: steps/greynoise/gnql-query/manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,834 @@
...
+  description: 'GNQL (GreyNoise Query Language) is a domain-specific query language
+    that uses Lucene deep under the hood. GNQL aims to enable GreyNoise Enterprise
+    and Research users to make complex and one-off queries against the GreyNoise dataset
+    as new business cases arise. GNQL is built with self-defeat and fully featured
+    product lines in mind. If we do our job correctly, each individual GNQL query
+    that brings our users and customers sufficient value will eventually be transitioned
+    into it''s own individual offering.
+
+    More info: https://developer.greynoise.io/reference/gnql-1#gnqlquery-1'
+  name: gnql_query
```
Comment (hagaishapira) [inline steps/greynoise/gnql-query/manifest.yaml:31]:
```suggestion
  name: greynoise_gnql_query
```

File: steps/greynoise/ip-context/manifest.yaml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    IP metadata (network owner, ASN, reverse DNS pointer, country), associated actors,
+    activity tags, and raw port scan and web request information.
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#noisecontextip-1'
+  name: ip_context
+  type: remediation
+  vendor: greynoise
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/ip-context/manifest.yaml:22]:
```suggestion
- description: JSON object containing the response from the GreyNoise API.
```

File: steps/greynoise/ip-quick-check/manifest.yaml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    has been observed scanning or attacking devices across the Internet.
+
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#quickcheck-1'
+  name: ip_quick_check
+  type: remediation
+  vendor: greynoise
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/ip-quick-check/manifest.yaml:22]:
```suggestion
- description: JSON object containing the response from the GreyNoise API.
```

File: steps/greynoise/multi-ip-quick-check/manifest.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    step is functionality identical to the "IP Quick Check" step, except it processes
+    more than one checks simultaneously.
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#postquickcheckmulti'
+  name: multi_ip_quick_check
+  type: remediation
+  vendor: greynoise
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/multi-ip-quick-check/manifest.yaml:23]:
```suggestion
- description: JSON object containing the response from the GreyNoise API.
```

File: steps/greynoise/riot-ip-lookup/manifest.yaml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    The collection of IPs in RIOT is continually curated and verified to provide accurate
+    results.
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#riotip'
+  name: riot_ip_lookup
+  type: remediation
+  vendor: greynoise
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/riot-ip-lookup/manifest.yaml:23]:
```suggestion
- description: JSON object containing the response from the GreyNoise API.
```

File: steps/greynoise/gnql-stats/manifest.yaml [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+  type: remediation
+  vendor: greynoise
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/gnql-stats/manifest.yaml:27]:
```suggestion
- description: JSON object containing the response from the GreyNoise API.
```

File: steps/greynoise/ip-context/manifest.yaml [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+integrations:
+- greynoise_token
+kind: Step
+metadata:
+  description: 'Get more information about a given IP address. Returns time ranges,
+    IP metadata (network owner, ASN, reverse DNS pointer, country), associated actors,
+    activity tags, and raw port scan and web request information.
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#noisecontextip-1'
+  name: ip_context
```
Comment (hagaishapira) [inline steps/greynoise/ip-context/manifest.yaml:17]:
```suggestion
  name: greynoise_ip_context
```

File: steps/greynoise/ip-quick-check/manifest.yaml [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+integrations:
+- greynoise_token
+kind: Step
+metadata:
+  description: 'Check whether a given IP address is "Internet background noise", or
+    has been observed scanning or attacking devices across the Internet.
+
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#quickcheck-1'
+  name: ip_quick_check
```
Comment (hagaishapira) [inline steps/greynoise/ip-quick-check/manifest.yaml:17]:
```suggestion
  name: greynoise_ip_quick_check
```

File: steps/greynoise/multi-ip-quick-check/manifest.yaml [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+- greynoise_token
+kind: Step
+metadata:
+  description: 'Check whether a set of IP addresses are "Internet background noise",
+    or have been observed scanning or attacking devices across the Internet. This
+    step is functionality identical to the "IP Quick Check" step, except it processes
+    more than one checks simultaneously.
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#postquickcheckmulti'
+  name: multi_ip_quick_check
```
Comment (hagaishapira) [inline steps/greynoise/multi-ip-quick-check/manifest.yaml:18]:
```suggestion
  name: greynoise_multi_ip_quick_check
```

File: steps/greynoise/multi-ip-quick-check/manifest.yaml [hunk 13]
```diff
@@ -0,0 +1,38 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: IP addresses to query
```
Comment (hagaishapira) [inline steps/greynoise/multi-ip-quick-check/manifest.yaml:3]:
```suggestion
- description: IP addresses to query.
```

File: steps/greynoise/riot-ip-lookup/manifest.yaml [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+- greynoise_token
+kind: Step
+metadata:
+  description: 'RIOT identifies IPs from known benign services and organizations that
+    commonly cause false positives in network security and threat intelligence products.
+    The collection of IPs in RIOT is continually curated and verified to provide accurate
+    results.
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#riotip'
+  name: riot_ip_lookup
```
Comment (hagaishapira) [inline steps/greynoise/riot-ip-lookup/manifest.yaml:18]:
```suggestion
  name: greynoise_riot_ip_lookup
```

File: steps/virustotal/comment-on-resource/manifest.yaml [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+  type: string
+integrations:
+- virustotal_token
+kind: Step
+metadata:
+  description: 'Post comment for a file or URL.
+
+
+    More info: https://developers.virustotal.com/reference#comments-put'
+  name: comment_on_resource
```
Comment (hagaishapira) [inline steps/virustotal/comment-on-resource/manifest.yaml:22]:
```suggestion
  name: virustotal_comment_on_resource
```

File: steps/virustotal/comment-on-resource/manifest.yaml [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+  type: remediation
+  vendor: virustotal
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the VirusTotal API.
```
Comment (hagaishapira) [inline steps/virustotal/comment-on-resource/manifest.yaml:27]:
```suggestion
- description: JSON object containing the response from the VirusTotal API.
```

File: steps/virustotal/rescan/manifest.yaml [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+    result.
+
+
+    More info: https://developers.virustotal.com/reference#file-rescan'
+  name: rescan
+  type: remediation
+  vendor: virustotal
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the VirusTotal API.
```
Comment (hagaishapira) [inline steps/virustotal/rescan/manifest.yaml:26]:
```suggestion
- description: JSON object containing the response from the VirusTotal API.
```

File: steps/virustotal/retrieve-report/manifest.yaml [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+  description: 'Retrieve file scan reports.
+
+
+    More info: https://developers.virustotal.com/reference#api-responses'
+  name: retrieve_report
+  type: remediation
+  vendor: virustotal
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the VirusTotal API.
```
Comment (hagaishapira) [inline steps/virustotal/retrieve-report/manifest.yaml:28]:
```suggestion
- description: JSON object containing the response from the VirusTotal API.
```

File: steps/virustotal/retrieve-scan-url-report/manifest.yaml [hunk 19]
```diff
@@ -0,0 +1,59 @@
...
+  description: 'Retrieve URL scan reports.
+
+
+    More info: https://developers.virustotal.com/reference#url-report'
+  name: retrieve_scan_url_report
+  type: remediation
+  vendor: virustotal
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the VirusTotal API.
```
Comment (hagaishapira) [inline steps/virustotal/retrieve-scan-url-report/manifest.yaml:35]:
```suggestion
- description: JSON object containing the response from the VirusTotal API.
```

File: steps/virustotal/scan-file/manifest.yaml [hunk 20]
```diff
@@ -0,0 +1,41 @@
...
+    in size you must request a special upload URL using the "upload_url" step.
+
+
+    More info: https://developers.virustotal.com/reference#file-scan'
+  name: scan_file
+  type: remediation
+  vendor: virustotal
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the VirusTotal API.
```
Comment (hagaishapira) [inline steps/virustotal/scan-file/manifest.yaml:30]:
```suggestion
- description: JSON object containing the response from the VirusTotal API.
```

File: steps/virustotal/scan-url/manifest.yaml [hunk 21]
```diff
@@ -0,0 +1,32 @@
...
+  description: 'Scan an URL.
+
+
+    More info: https://developers.virustotal.com/reference#url-scan'
+  name: scan_url
+  type: remediation
+  vendor: virustotal
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the VirusTotal API.
```
Comment (hagaishapira) [inline steps/virustotal/scan-url/manifest.yaml:21]:
```suggestion
- description: JSON object containing the response from the VirusTotal API.
```

File: steps/virustotal/upload-url/manifest.yaml [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+kind: Step
+metadata:
+  description: "Get a URL for uploading files larger than 32MB. This step requires\
+    \ additional privileges on VirusTotal side. \n\nMore info: https://developers.virustotal.com/reference#file-scan-upload-url"
+  name: upload_url
+  type: remediation
+  vendor: virustotal
+  version: 1.0.0
+outputs:
+- description: Object containing an the response from the VirusTotal API.
```
Comment (hagaishapira) [inline steps/virustotal/upload-url/manifest.yaml:14]:
```suggestion
- description: JSON object containing the response from the VirusTotal API.
```

File: steps/virustotal/upload-url/manifest.yaml [hunk 23]
```diff
@@ -0,0 +1,20 @@
+apiVersion: stackpulse.io/v1
+envs: []
+integrations:
+- virustotal_token
+kind: Step
+metadata:
+  description: "Get a URL for uploading files larger than 32MB. This step requires\
+    \ additional privileges on VirusTotal side. \n\nMore info: https://developers.virustotal.com/reference#file-scan-upload-url"
+  name: upload_url
```
Comment (hagaishapira) [inline steps/virustotal/upload-url/manifest.yaml:9]:
```suggestion
  name: virustotal_upload_url
```

File: steps/virustotal/upload-url/manifest.yaml [hunk 24]
```diff
@@ -0,0 +1,20 @@
+apiVersion: stackpulse.io/v1
+envs: []
+integrations:
+- virustotal_token
+kind: Step
+metadata:
+  description: "Get a URL for uploading files larger than 32MB. This step requires\
```
Comment (hagaishapira) [inline steps/virustotal/upload-url/manifest.yaml:7]: how are those slashes gonna look in the planner ?

File: steps/virustotal/scan-url/manifest.yaml [hunk 25]
```diff
@@ -0,0 +1,32 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: The URL that should be scanned
```
Comment (hagaishapira) [inline steps/virustotal/scan-url/manifest.yaml:3]:
```suggestion
- description: The URL that should be scanned.
```

File: steps/greynoise/gnql-query/manifest.yaml [hunk 26]
```diff
@@ -0,0 +1,834 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: GNQL query string
```
Comment (hagaishapira) [inline steps/greynoise/gnql-query/manifest.yaml:3]:
```suggestion
- description: GNQL query string.
```

File: steps/greynoise/gnql-query/manifest.yaml [hunk 27]
```diff
@@ -0,0 +1,834 @@
...
+  example: raw_data.ja3.fingerprint:795bc7ce13f60d61e9ac03611dd36d90
+  name: QUERY
+  required: true
+  type: string
+- description: Maximum amount of results to grab
```
Comment (hagaishapira) [inline steps/greynoise/gnql-query/manifest.yaml:8]:
```suggestion
- description: Maximum amount of results to grab.
```

File: steps/greynoise/gnql-query/manifest.yaml [hunk 28]
```diff
@@ -0,0 +1,834 @@
...
+  example: '10000'
+  name: SIZE
+  required: false
+  type: int
+- description: Scroll token to paginate through results
```
Comment (hagaishapira) [inline steps/greynoise/gnql-query/manifest.yaml:13]:
```suggestion
- description: Scroll token to paginate through results.
```

File: steps/greynoise/gnql-stats/manifest.yaml [hunk 29]
```diff
@@ -0,0 +1,250 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: GNQL query string
```
Comment (hagaishapira) [inline steps/greynoise/gnql-stats/manifest.yaml:3]:
```suggestion
- description: GNQL query string.
```

File: steps/greynoise/gnql-stats/manifest.yaml [hunk 30]
```diff
@@ -0,0 +1,250 @@
...
+  example: raw_data.ja3.fingerprint:795bc7ce13f60d61e9ac03611dd36d90
+  name: QUERY
+  required: true
+  type: string
+- description: Number of top aggregates to grab
```
Comment (hagaishapira) [inline steps/greynoise/gnql-stats/manifest.yaml:8]:
```suggestion
- description: Number of top aggregates to grab.
```

File: steps/greynoise/ip-context/manifest.yaml [hunk 31]
```diff
@@ -0,0 +1,67 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: IP address to query
```
Comment (hagaishapira) [inline steps/greynoise/ip-context/manifest.yaml:3]:
```suggestion
- description: IP address to query.
```

File: steps/greynoise/ip-quick-check/manifest.yaml [hunk 32]
```diff
@@ -0,0 +1,30 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: IP address to query
```
Comment (hagaishapira) [inline steps/greynoise/ip-quick-check/manifest.yaml:3]:
```suggestion
- description: IP address to query.
```

File: steps/greynoise/riot-ip-lookup/manifest.yaml [hunk 33]
```diff
@@ -0,0 +1,37 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: IP address to query
```
Comment (hagaishapira) [inline steps/greynoise/riot-ip-lookup/manifest.yaml:3]:
```suggestion
- description: IP address to query.
```

File: steps/virustotal/comment-on-resource/manifest.yaml [hunk 34]
```diff
@@ -0,0 +1,34 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: Either an md5/sha1/sha256 hash of the file you want to review or the
+    URL itself that you want to comment on
```
Comment (hagaishapira) [inline steps/virustotal/comment-on-resource/manifest.yaml:4]:
```suggestion
- description: Either an md5/sha1/sha256 hash of the file you want to review or the URL itself that you want to comment on.
```

File: steps/virustotal/comment-on-resource/manifest.yaml [hunk 35]
```diff
@@ -0,0 +1,34 @@
...
+  example: 8ebc97e05c8e1073bda2efb6f4d00ad7e789260afa2c276f0c72740b838a0a93
+  name: RESOURCE
+  required: true
+  type: string
+- description: The comment's text
```
Comment (hagaishapira) [inline steps/virustotal/comment-on-resource/manifest.yaml:9]:
```suggestion
- description: The comment text.
```

File: steps/virustotal/retrieve-scan-url-report/manifest.yaml [hunk 36]
```diff
@@ -0,0 +1,59 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: A URL for which you want to retrieve the most recent report. You may
+    also specify a scan_id (sha256-timestamp as returned by the URL submission API)
+    to access a specific report.
+  example: http://virustotal.com/
+  name: RESOURCE
+  required: true
+  type: string
+- description: Return additional information about the file
```
Comment (hagaishapira) [inline steps/virustotal/retrieve-scan-url-report/manifest.yaml:10]:
```suggestion
- description: Return additional information about the file.
```

File: steps/virustotal/retrieve-scan-url-report/manifest.yaml [hunk 37]
```diff
@@ -0,0 +1,59 @@
...
+  type: string
+integrations:
+- virustotal_token
+kind: Step
+metadata:
+  description: 'Retrieve URL scan reports.
+
+
+    More info: https://developers.virustotal.com/reference#url-report'
+  name: retrieve_scan_url_report
```
Comment (hagaishapira) [inline steps/virustotal/retrieve-scan-url-report/manifest.yaml:30]:
```suggestion
  name: virustotal_retrieve_scan_url_report
```

File: steps/virustotal/scan-file/manifest.yaml [hunk 38]
```diff
@@ -0,0 +1,41 @@
+apiVersion: stackpulse.io/v1
+envs:
+- description: File to be scanned, in base64
```
Comment (hagaishapira) [inline steps/virustotal/scan-file/manifest.yaml:3]:
```suggestion
- description: File to be scanned, in base64.
```

File: steps/virustotal/scan-file/manifest.yaml [hunk 39]
```diff
@@ -0,0 +1,41 @@
...
+  example: ZXhhbXBsZSBmaWxlIGNvbnRlbnQ=
+  name: FILE
+  required: true
+  type: string
+- description: Name of the file. This field is mandatory if FILE is provided
```
Comment (hagaishapira) [inline steps/virustotal/scan-file/manifest.yaml:8]:
```suggestion
- description: Name of the file.
```

File: steps/virustotal/scan-file/manifest.yaml [hunk 40]
```diff
@@ -0,0 +1,41 @@
...
+metadata:
+  description: 'This step allows you to send a file for scanning with VirusTotal.
+    Before performing your submissions VirusTotal encourage you to retrieve the latest
+    report on the file, if it is recent enough you might want to save time and bandwidth
+    by making use of it. File size limit is 32MB, in order to submit files up to 200MB
+    in size you must request a special upload URL using the "upload_url" step.
+
+
+    More info: https://developers.virustotal.com/reference#file-scan'
+  name: scan_file
```
Comment (hagaishapira) [inline steps/virustotal/scan-file/manifest.yaml:25]:
```suggestion
  name: virustotal_scan_file
```

File: steps/virustotal/scan-url/manifest.yaml [hunk 41]
```diff
@@ -0,0 +1,32 @@
...
+  type: string
+integrations:
+- virustotal_token
+kind: Step
+metadata:
+  description: 'Scan an URL.
+
+
+    More info: https://developers.virustotal.com/reference#url-scan'
+  name: scan_url
```
Comment (hagaishapira) [inline steps/virustotal/scan-url/manifest.yaml:16]:
```suggestion
  name: virustotal_scan_url
```

**Review by hagaishapira — CHANGES_REQUESTED** [review]

File: steps/greynoise/gnql-query/manifest.yaml (hunk 2 above)
Comment (InbarPerry) [reply steps/greynoise/gnql-query/manifest.yaml:1]: i think that the yaml library are reordering the yaml alphabetically

File: steps/virustotal/upload-url/manifest.yaml (hunk 24 above)
Comment (InbarPerry) [reply steps/virustotal/upload-url/manifest.yaml:7]: I don't know why it generated it like that. I have no idea how it would look.

Comment (InbarPerry) [reply steps/virustotal/upload-url/manifest.yaml:7]: should look fine

File: steps/greynoise/gnql-stats/manifest.yaml [hunk 42]
```diff
@@ -0,0 +1,250 @@
...
+  required: true
+  example: raw_data.ja3.fingerprint:795bc7ce13f60d61e9ac03611dd36d90
+- name: COUNT
+  description: Number of top aggregates to grab.
+  type: int
+  required: false
+  example: '10000'
+outputs:
+- name: api_object
+  description: A JSON object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/gnql-stats/manifest.yaml:26]:
```suggestion
  description: A JSON object containing the response from the GreyNoise API.
```

File: steps/greynoise/ip-quick-check/manifest.yaml [hunk 43]
```diff
@@ -0,0 +1,30 @@
...
+  type: remediation
+envs:
+- name: IP
+  description: IP address to query.
+  type: string
+  required: true
+  example: 1.1.1.1
+outputs:
+- name: api_object
+  description: A JSON object containing an the response from the GreyNoise API.
```
Comment (hagaishapira) [inline steps/greynoise/ip-quick-check/manifest.yaml:21]:
```suggestion
  description: A JSON object containing the response from the GreyNoise API.
```

Comment (hagaishapira) [reply steps/greynoise/ip-quick-check/manifest.yaml:21]: This pretty much continues so it's probably in your script @InbarPerry

File: steps/greynoise/multi-ip-quick-check/manifest.yaml [hunk 44]
```diff
@@ -0,0 +1,38 @@
...
+    or have been observed scanning or attacking devices across the Internet. This
+    step is functionality identical to the "IP Quick Check" step, except it processes
+    more than one checks simultaneously.
+
+    More info: https://developer.greynoise.io/reference/ip-lookup-1#postquickcheckmulti'
+  vendor: greynoise
+  type: remediation
+envs:
+- name: IP_LIST
+  description: IP addresses to query.
```
Comment (hagaishapira) [inline steps/greynoise/multi-ip-quick-check/manifest.yaml:16]:
```suggestion
  description: IP addresses to query in a JSON array form.
```

**Review by hagaishapira — APPROVED** [review]
Fix the recurring api_object description typo and the other comment and MWG

**Pull request merged**